low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -797,38 +756,6 @@ class SSPLHealthView(object):
                 store.put(system[singleton_realstorencl.FAULT_KEY],
                     singleton_realstorencl.faults_persistent_cache)
 
-    def build_encl_hw_specifics_instances(self):
-        fan_module_list = []
-        sideplane_list = []
-        chassis_midplane_list = []
-        enclosure_data = self.get_realstor_show_data(self.encl_hw_specifics_api)
```

Comment:
Remove "self.encl_hw_specifics_api" defined variable from line no 94 and
remove line no 794 to 796 and 804 to 805, that code is unused in this file.

Reply:
> [ if fru == 'configuration': ]
 should we leave this code for future use-case?
 or I should remove this code?

Reply:
We will add it when required. do not need now.

Reply:
Done